experimental/PropertyDDS/packages/property-binder/src/internal/property_element.js

```diff
@@ -334,9 +334,15 @@ class PropertyElement {
       return this;
     }
 
-    if (typeof in_child === 'string' || typeof in_child === 'number') {
+    let isPathString = false;
```

Comment:
You should adapt the comment above, to state that it either takes a single quoted path segment or a tokenized path as an array.

Reply:
The whole section that compares the tokens to special characters below should only be enabled if a isPathString is true, as it would give ambiguous results on a tokenized path.